index.js

```diff
@@ -14,6 +14,8 @@ function Attempt(instance, end) {
   var command = [];
   command.push('/usr/bin/sudo');
   command.push('-n');
+  // Preserve user environment.
```

Comment:
Can you make it a colon here instead of a full stop? Then it's clear that it refers to the `-E` option not the line above. Longer paragraph comments or comments after a semicolon can use a full stop.